### PR TITLE
Remove error part from union return types and remove included record params rendering through flow model

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java [hunk 1]
```diff
@@ -626,8 +639,14 @@ public void visit(FunctionCallExpressionNode functionCallExpressionNode) {
...
-            functionSymbol.typeDescriptor().params().ifPresent(params -> nodeBuilder.properties().functionArguments(
-                    functionCallExpressionNode.arguments(), params, documentationMap, functionSymbol.external()));
+            Optional<List<ParameterSymbol>> funcParams = functionSymbol.typeDescriptor().params();
+            if (funcParams.isPresent()) {
+                List<ParameterSymbol> params = funcParams.get().stream()
+                        .filter(p -> p.paramKind() != ParameterKind.INCLUDED_RECORD)
+                        .toList();
+                nodeBuilder.properties().functionArguments(
+                        functionCallExpressionNode.arguments(), params, documentationMap, functionSymbol.external());
+            }
```
Comment (KavinduZoysa) [inline flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java:649]: Can't we use `handleActionCallsParams`?

**Review by hasithaa — APPROVED** [review]

File: flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/diagram_generator/config/remote_action_call-http-post1.json [hunk 2]
```diff
@@ -239,16 +239,6 @@
...
-          "params": {
-            "metadata": {
-              "label": "params",
-              "description": "The query parameters"
-            },
-            "valueType": "EXPRESSION",
-            "optional": false,
-            "editable": true,
-            "advanced": false
-          },
```
Comment (nipunayf) [inline flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/diagram_generator/config/remote_action_call-http-post1.json:251]: Why r we removing the params property?

File: flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java (hunk 1 above)
Comment (nipunayf) [reply flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java:649]: +1

Comment (LakshanWeerasinghe) [reply flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java]: updated in [commit](https://github.com/ballerina-platform/ballerina-dev-tools/pull/465/commits/d81e4d04436716c0ba7d36547223ba5181fd659c).

**Review by nipunayf — APPROVED** [review]

**Pull request merged**